SoftLayer/CLI/virt/create_options.py

```diff
@@ -25,35 +25,61 @@ def cli(env):
     # Datacenters
     datacenters = [dc['template']['datacenter']['name']
                    for dc in result['datacenters']]
+    datacenters = sorted(datacenters)
+
     table.add_row(['datacenter',
                    formatting.listing(datacenters, separator='\n')])
 
-    # CPUs
-    standard_cpu = [x for x in result['processors']
-                    if not x['template'].get(
-                        'dedicatedAccountHostOnlyFlag', False)]
-
-    ded_cpu = [x for x in result['processors']
-               if x['template'].get('dedicatedAccountHostOnlyFlag',
-                                    False)]
-
-    def add_cpus_row(cpu_options, name):
-        """Add CPU rows to the table."""
-        cpus = []
-        for cpu_option in cpu_options:
-            cpus.append(str(cpu_option['template']['startCpus']))
+    bal_flavors = [str(x['flavor']['keyName']) for x in result['flavors']
```

Comment:
You could put flavor extraction into a separate method and generalize it to avoid duplication here.

Reply:
Will do.